### DISCUSSION TITLE
new user - black screen after logging in on one PC only

### DISCUSSION THREAD

**Discussion by mac1-131** [rustdesk/rustdesk#8107] [Q&A] [answered]
Hi I have one PC that when it remotes in to another PC, it logs in successfully but its screen is always black.

This happens no matter what remote PC I log in to. The mouse on the remote PC moves around fine, I just cannot see the screen as it is all black.

When I remote IN to this bad PC, everything is fine. When I initiate a connection out, screen is always black.

I suspect something display driver related, can anyone help? Display driver is Intel Q35 Express Chipset Family (Microsoft Corporation - WDDM 1.0).

thanks

![image](https://github.com/rustdesk/rustdesk/assets/57259400/6c31041b-711c-4b71-ad25-e33d9a19ce66)

**Comment by 21pages** [comment]
Enable `Show quality monitor` and try different `Codec`

![1716292977127](https://github.com/rustdesk/rustdesk/assets/14891774/e429f3db-5b14-4c92-96cf-a50698213d91)

**Reply by mac1-131** [reply]
thanks, I tried your suggestion no change, still black screen

**Reply by 21pages** [reply]
Ensure you are using latest nightly, and record a video, then check whether the mp4 or webm file is good.

![image](https://github.com/rustdesk/rustdesk/assets/14891774/14f1a9ca-c399-4831-8f1b-8f46de160049)

![image](https://github.com/rustdesk/rustdesk/assets/14891774/28b5c2ec-f71a-475f-98db-ba16132b21cb)

**Reply by mac1-131** [reply]
OK, did that here is the result.

None of the media players on my PC would play the video. (Windows 10) The MS media player took me to the microsoft store to download the AV1 codec but even after that, it would not play.

One of the file options to open the video was Firefox. **Firefox played the video fine, while all of the media players on my PC would not play the video.**

![image](https://github.com/rustdesk/rustdesk/assets/57259400/edfc665f-0598-4074-8106-71e0caf0191c)

![image](https://github.com/rustdesk/rustdesk/assets/57259400/267aeb11-1885-4c95-9ae6-3f1d87fab177)

**Reply by rustdesk** [reply]
> thanks, I tried your suggestion no change, still black screen

you really tried out the other codecs? please record a video for your operations. also try out 1.2.3-2, 1.1.9, and the sciter version (the exe file with name containing sciter).

**Reply by mac1-131** [reply]
rustdesk-1.2.3-2-x86-sciter.exe does work, but rustdesk-1.2.3-2-x86_64.exe does not

and yes I tried all the codecs, same result (VP9, VP8, AV1, H264, H265) was able to record good video on most 
of those, but still black screen on rustdesk

I sent log to 21pages

thanks for the help

[c1395322888_20240524160249_VP9.webm](https://github.com/rustdesk/rustdesk/assets/57259400/5444ab9e-5c33-45f9-9cff-3f243d12242d)

**Reply by rustdesk** [reply]
Thanks. I guess I know the problem, please wait for our update, this need some time. Please do not ask ETA, :(.

**Reply by mac1-131** [reply]
no problem I can use sciter version and happy to test for you when you are ready. thank you.

**Reply by rustdesk** [reply]
Please try out the new nightly build, please verify if it does work first.

If it does not work, please uncheck below (controlling side) and reconnect.

![image](https://github.com/rustdesk/rustdesk/assets/71636191/f7701b3e-9e33-4d62-9941-e85eb376e75a)

**Reply by mac1-131** [reply]
does not work with box checked

works fine with box un-checked!!!

**Reply by rustdesk** [reply, marked as answer]
Thanks, that's expected.

**Comment by assecajonsoen89966** [comment]
I ran into black screen right after login on my laptop a few months back. I could move the mouse but nothing else loaded. Safe mode worked fine and that is how I started fixing it. I uninstalled the graphics driver from Device Manager and rebooted which allowed Windows to reinstall the default driver. That cleared the problem instantly and then I updated the driver from the manufacturer’s site. Another tip if it still happens is using Task Manager to restart Windows Explorer because many times the shell just fails to load. After locking down the drivers I also checked activation since missing updates can cause weird issues. I bought a license through **Softwarekey4u** and since then my updates run without interruptions. If your PC keeps showing black screens after login it is usually display driver or corrupted system file related.
